tests/index.test.ts

```diff
@@ -19,27 +23,34 @@ const quantity = 10
 const metric = 'dotproduct'
 const vectors = generateVectors(dimensions, quantity)
 const vectorsWithSparseValues = generateVectors(dimensions, quantity, true)
+const client: PineconeClient = new PineconeClient()
+
+beforeEach(() => {
+  jest.setTimeout(100000)
```

Comment:
Is this the maximum amount of time per all tests, 100 seconds?

Reply:
I think it's the maximum time for an individual test. Some of the tests are doing time-consuming things like creating a new index, which involves waiting a bit before it is ready. In reality, most tests do not take anywhere near that long.